packages-exp/auth-exp/test/integration/smoke_test.test.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect } from 'chai';
+
+import { signInAnonymously } from '@firebase/auth-exp/index.browser';
+import { OperationType } from '@firebase/auth-types-exp';
+
+import { describeIntegration } from '../helpers/integration/helpers';
+
+describeIntegration('smoke test', auth => {
```

Comment:
Future you will probably regret doing this. It breaks all UI test runners and certain lint rules (such as prohibiting describe.only).

Can you follow the pattern we use in Firestore? https://github.com/firebase/firebase-js-sdk/blob/master/packages/firestore/lite/test/integration.test.ts#L145

Reply:
So the issue with that pattern is we need the cleanup code to always run, even if the test fails. So to do that, each `describe` needs an `after`. I was trying to avoid code duplication as much as possible.

I've switched this up to have a bit of a compromise.